asconfig.release.json

```diff
@@ -53,7 +53,7 @@
             },
             {
                 "name": "R3::VERSION",
-                "value": "\"1.1.1\""
+                "value": "\"1.2.1\""
```

Comment:
If we follow semver and make a minor bump, shouldn't the patch version be zero? (for 1.2.0)

Reply:
I created a previous "release 1.2.0" which had bugs, and I didn't release. (See release drafts.)